counterexamples/admins/locality/invalid-max-length-limit-is-alpha2.json

```diff
@@ -0,0 +1,88 @@
+{
+    "type": "Feature",
+    "properties": {
+      "theme": "admins",
+      "type": "locality",
+      "version": 0,
+      "updateTime": "2023-02-22T23:55:01-08:00",
+      "subType": "administrativeLocality",
+      "localityType": "country",
+      "adminLevel": 1,
+      "isoCountryCodeAlpha2": "USA",
+      "defaultLanguage": "english",
+      "drivingSide": "right",
+      "contextId": "northAmericaId",
+      "names": {
+        "common": [
+          {
+            "language": "local",
+            "value": "United States"
+          },
+          {
+            "language": "en",
+            "value": "United States"
+          },
+          {
+            "language": "es",
+            "value": "Estados Unidos"
+          }
+        ],
+        "official": [
+          {
+            "language": "local",
+            "value": "United States of America"
+          }
+        ],
+        "short": [
+          {
+            "language": "local",
+            "value": "US"
+          },
+          {
+            "language": "en",
+            "value": "US"
+          },
+          {
+            "language": "en",
+            "value": "U.S."
+          }
+        ],
+        "alternate": [
+          {
+            "language": "local",
+            "value": "States"
+          }
+        ]
+      },
```

Comment:
1. [ ] This is the **old** names schema. If it validates, that would be surprising. Something is wrong, can you find out what?
2. [ ] The `names` is ultra-long but not the main point of this counterexample. Can you shorten it to just the primary name or even drop the `names` property altogether to make a simpler counterexample?

Reply:
Removed the names property altogether since it doesn't make sense with these counterexamples.
upon looking at the `names` property schema definition, the counterexamples do use the old names schema but isn't that fine for counterexamples since it's expected to fail?

Reply:
Kind of yes, but mostly no.

The reason no, is that most of those counterexamples aren't meant to test the that aspect, they're supposed to test something else. So basically there's a false failure signal that could hide the true failure signal, if that makes sense.